src/ComboControls.ts

```diff
@@ -537,8 +533,9 @@ export default class ComboControls extends EventDispatcher {
     let targetDistance = offsetVector.length();
 
     // half of the fov is center to top of screen
-    if (camera instanceof PerspectiveCamera) {
-      targetDistance *= Math.tan(((camera.fov / 2) * Math.PI) / 180);
+    // @ts-ignore
```

Comment:
Does it give you a warning without this?

Reply:
Yes, because `.isPerspectiveCamera` property does not exist on type `OrthographicCamera`

Reply:
Aha true